sandbox/src/components/charts/HighchartFactory.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import CodeExample from "./CodeExample";
+import highchartConst from "./highchartConst";
+import history from "../../history";
+import { highchart } from "../UrlConst";
+
+export default function Highchart(props) {
+  let { url } = props.match.params;
+  console.log("history", history);
+  if (url || history.location.pathname === "/react-highcharts/") {
```

Comment:
Find a better way than this:  history.location.pathname === "/react-highcharts/"?